043-SQLModernization/Student/Challenge00.md

```diff
@@ -0,0 +1,53 @@
+# Challenge 0 - Setup
+
+**[Home](../README.md)** - [Next Challenge >](./Challenge01.md)
+
+## Pre-requisites 
+
+An Azure subscription is required that can deploy Azure SQL resources (Azure SQL Database and Azure SQL Managed Instance), Virtual Machines.
+
+## Notes & Guidance
```

Comment:
"Notes & Guidance" is the header for the Coach guide template.  I observed you used this for the first section header in each of the student guide MD files.

All instances of "Notes & Guidance" in the student guide should be changed to "Introduction" to match our template.

Reply:
done, will be in PR